addons/web/__manifest__.py

```diff
@@ -381,6 +381,8 @@
             ('remove', 'web/static/src/core/debug/debug_menu.js'),
             ('remove', 'web/static/src/core/file_viewer/file_viewer.dark.scss'),
             ('remove', 'web/static/src/core/emoji_picker/emoji_data.js'),
+            'web/static/src/core/commands/default_providers.js',
```

Comment:
to check

Reply:
This is a comment to investigate in an other task